src/services/agentDistributionService.ts

```
@@ -0,0 +1,43 @@
+
+      logger.debug('Wallet generated successfully', {
+        address: response.data.address,
+      });
+
+      return response.data;
+    } catch (error) {
+      logger.error('Failed to generate wallet', { error });
+      throw new Error('Failed to generate funding pool address');
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)